python/ray/experimental/state/state_cli.py

```diff
@@ -50,53 +51,53 @@ def list_state_cli_group(ctx):
 @click.pass_context
 def actors(ctx):
     url = ctx.obj["api_server_url"]
-    pprint(list_actors(url))
+    pprint(list_actors(api_server_url=url, _print_api_stats=True))
```

Comment:
should we make an option to suppress the warning?

Reply:
I think we can support env var once there are people who ask for it?